approved/v93k/prb1.tf

```diff
@@ -1778,24 +1764,10 @@ test_flow
     {
     }
     print_dl("Test unless_ran");
-    run_and_branch(erase_all_33_864CE8F)
-    then
-    {
-      @ERASE_RAN_3_864CE8F_RAN = 1;
-    }
-    else
-    {
-      @ERASE_RAN_3_864CE8F_RAN = 1;
-    }
-    run_and_branch(erase_all_34_864CE8F)
-    then
-    {
-      @ERASE_RAN_4_864CE8F_RAN = 1;
-    }
-    else
-    {
-      @ERASE_RAN_4_864CE8F_RAN = 1;
-    }
+    run(erase_all_33_864CE8F);
+    @ERASE_RAN_3_864CE8F_RAN = 1;
+    run(erase_all_34_864CE8F);
+    @ERASE_RAN_4_864CE8F_RAN = 1;
```

Comment:
New implementation of if_ran, set the flag immediately after the test has run rather than affecting the test itself.